zksync_sdk/zksync.py

```diff
@@ -0,0 +1,67 @@
+from eth_account.signers.base import BaseAccount
+from web3 import Web3
+
+from zksync_sdk.contract_utils import erc20_abi, zk_sync_abi
+
+MAX_ERC20_APPROVE_AMOUNT = 115792089237316195423570985008687907853269984665640564039457584007913129639935
```

Comment:
I think there should be a comment on where these large numbers come from (how they were evaluated)